pkl-codegen-java/src/test/kotlin/org/pkl/codegen/java/JavaCodeGeneratorTest.kt

```diff
@@ -775,6 +775,55 @@ class JavaCodeGeneratorTest {
       .isEqualToResourceFile("Inheritance.jva")
   }
 
+  @Test
+  fun `stateless classes`() {
```

Comment:
[nit] "empty" feels like a better word for this

```suggestion
  fun `empty classes`() {
```

Reply:
Ah, gotcha